package.json

```diff
@@ -36,7 +36,8 @@
     "readable-stream": "1.1.13"
   },
   "devDependencies": {
-    "aegir": "^3.2.0"
+    "aegir": "^3.2.0",
+    "ipfs": "^0.14.1"
```

Comment:
This is not a good idea, this adds the dependency to all the repos requiring it even though they might not need it